keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -586,8 +589,13 @@ feature cannot be used. Pods using the new `VolumeSource` combined with a not
 supported container runtime version will fail to run on the node, because the
 `Mount.host_path` field is not set for those mounts.
 
-For security reasons, volume mounts should set the [`noexec`] and `ro`
-(read-only) options by default.
+For the most conservative security approach, image volumes should be mounted by
+runtimes with enabled `rro` (recursive read-only) and `noexec` flag. Runtimes
```

Comment:
Not specific to image volumes, but rro-by-default was rejected during the discussion in https://github.com/kubernetes/enhancements/pull/3858

Reply:
Would it be possible to override the `noexec` functionality? My use case is to use the images a build cash by stuffing a git repo in there which may have scripts etc where the expectation would be they operate as normal.

Reply:
Alright, I'll added back the `ro` option, but also made clear that it's up to the runtime to decide the details.

Reply:
@AkihiroSuda Is there still a compatibility concern in this case?  OCI volume source is a new feature, not GA yet, which can still have changes made to it.  Assuming the runtime supports `rro`, what would the compatibility concern be?  There is not yet a large corpus of existing workloads that depend on this feature.

Reply:
Practically no concern, but I expect all volumes to treat the `recursiveReadOnly` flag in the consistent way